tests/chart_tests/test_astronomer_cp_dp_feature.py

```diff
@@ -60,3 +86,56 @@ def test_astronomer_both_cp_dp_disabled(self, kube_version):
 
         dp_resources = self.filter_charts_by_component(charts, "dataplane")
         assert len(dp_resources) == 0
+
+    def test_components_have_correct_plane_labels(self, kube_version):
+        """Test that all components have the correct plane labels according to the component map."""
+        charts = render_chart(
```

Comment:
We should not change terminology from what we use everywhere else. This should be `docs = render_chart()`